index.bs

```diff
@@ -811,7 +811,7 @@ const dir_ref = current_project_dir;
 if (!dir_ref) return;
 
 // Now get a file reference by showing a file picker:
-const file_ref = await self.chooseFileSystemEntries({type: 'openFile'});
+const file_ref = await showOpenFilePicker();
```

Comment:
Was removing `self.` intentional? I think it was there to emphasize the call's receiver.

Reply:
Yeah, I'm not sure it's useful for the example purposes. Normally global APIs are used directly without `self.` or `this.` or `window.`, it's odd for this one to stand out.

Reply:
I think this is a better form for example code. It's more explicit, and it's more likely to work when copy-pasted into an arbitrary app.

Please add the `self.` back.

Reply:
> it's more likely to work when copy-pasted into an arbitrary app

How so? I find it more confusing for beginners, because one normally doesn't write `self.fetch` or `self.setTimeout` etc. - those are used as regular global APIs, and this one doesn't seem any different.

If you're referring to shadowing of variables, then things like `var self = this;` are very common in real-world code (pre-ES6), and would break the `self.showOpenFilePicker` example above, but wouldn't break direct `showOpenFilePicker` call.

Reply:
I've never in my life used `self`

Reply:
> I've never in my life used self

I've never in my life killed a rabbit. But I am pretty sure others do. I know because I've seen rabbit meat.